Goes with ../glibc-2.3.2/glibc-2.3.2-cygwin.patch

If you apply the above patch, but forget this one, you get a cryptic error
 .../bin/../x86_64-unknown-linux-gnu/sys-root/usr/lib64/: file not recognized: Is a directory
when linking with -lpthread, because the generated libpthread.so
is missing a filename

by dank@kegel.com, 2004 Mar 30


--- glibc-linuxthreads-2.3.2/linuxthreads/Makefile.old	Tue Mar 30 10:33:12 2004
+++ glibc-linuxthreads-2.3.2/linuxthreads/Makefile	Tue Mar 30 10:33:24 2004
@@ -114,7 +114,7 @@
 install: $(inst_libdir)/libpthread.so
 $(inst_libdir)/libpthread.so: $(common-objpfx)format.lds \
 			      $(objpfx)libpthread.so$(libpthread.so-version) \
-			      $(inst_libdir)/$(patsubst %,$(libtype.oS),\
+			      $(inst_libdir)/$(patsubst %,$(libtype.oST),\
 							$(libprefix)pthread) \
 			      $(+force)
 	(echo '/* GNU ld script';\
@@ -122,7 +122,7 @@
 	 echo '   the static library, so try that secondarily.  */';\
 	 cat $<; \
 	 echo 'GROUP ( $(slibdir)/libpthread.so$(libpthread.so-version)' \
-	      '$(libdir)/$(patsubst %,$(libtype.oS),$(libprefix)pthread)'\
+	      '$(libdir)/$(patsubst %,$(libtype.oST),$(libprefix)pthread)'\
 	      ')' \
 	) > $@.new
 	mv -f $@.new $@
